Cargo.toml

```diff
@@ -1,67 +1,35 @@
 [package]
+authors = ["MaidSafe Developers <dev@maidsafe.net>"]
+description = "Pairing threshold cryptography"
+edition = "2021"
+license = "GPL-3.0"
 name = "blsttc"
-version = "4.0.0"
-authors = [
-  "Vladimir Komendantskiy <komendantsky@gmail.com>",
-  "Andreas Fackler <AndreasFackler@gmx.de>",
-  "Peter van Nostrand <jnz@riseup.net>",
-  "Andrew Gross <andogro@gmail.com>",
-  "Nick Sanders <nsan1129@gmail.com>",
-  "Marc Brinkmann <git@marcbrinkmann.de>"
-]
-categories = [ "cryptography" ]
-keywords = [ "pairing", "threshold" ]
-license = "MIT/Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/maidsafe/blsttc"
-description = "Pairing threshold cryptography"
-edition = "2018"
+version = "4.0.0"
 
 [dependencies]
-byteorder = "1.3.4"
-ff = "0.6.0"
-group = "0.6.0"
+blst = { version = "0.3.7", features = ["portable"] }
+blstrs = { version = "0.4.2", features = ["portable"] }
+ff = "0.11.0"
+group = "0.11.0"
 hex_fmt = "0.3.0"
-log = "0.4.8"
-pairing = "0.16.0"
-rand = "0.7.3"
-rand_chacha = "0.2.2"
+pairing = "0.21.0"
+rand = "0.8.5"
+rand_chacha = "0.3.1"
+serde = { version = "1.0.104", features = ["derive"] }
```

Comment:
we should make serde optional.   (optional=true).    However, that can be done in a separate PR.

Reply:
Ok let's do it in a separate PR. To add a bit of extra context, historically serde has never been optional (from back to threshold_crypto days). I think it'll be good to have serde as optional in the future.